packages/components/src/interfaces/complex/InfoCards.ts

```diff
@@ -11,13 +11,11 @@ import { LINK_HREF_PATTERN } from "~/utils/validation"
 const SingleCardNoImageSchema = Type.Object({
   title: Type.String({
     title: "Title",
-    default: "This is the title of the card",
```

Comment:
The default field was meant to be there as a placeholder. Do we know why this occurs in first place? Because I remember other components do not have issues with default values

Reply:
the other components don't use `default` and the placeholders are derived from `apps/studio/src/components/PageEditor/constants.ts`

Not sure why this occur in the first place